File: src/src/Markify.Document/DocumentationOrganizerModule.fs
namespace Markify.Document

open System.Diagnostics.CodeAnalysis
open Markify.Core.Analyzers

open Ninject.Modules

[<ExcludeFromCodeCoverage>]
type DocumentationOrganizerModule() =
    inherit NinjectModule()

    override this.Load () =
        this.Bind<IDocumentationOrganizer>().To<BasicDocumentationOrganizer>() |> ignore

File: src/Application/Markify.Application/Services/DocumentationOrganizerModule.cs
using Markify.Domain.Document;
using Markify.Services.Document;
using Ninject.Modules;

namespace Markify.Application.Services
{ 
    public class DocumentationOrganizerModule : NinjectModule
    {
        public override void Load()
        {
            Bind<IDocumentOrganizer>().To<SimpleDocumentOrganizer>();
        }
    }
}